### PR TITLE
fixing license headers on source files

### PR DESCRIPTION
**Pull request by marcusthebrown**
@jodygarnett let me know if this addresses all your concerns

### CODE REVIEW

**Comment by marcusthebrown** [comment]
fixes #44

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {yyyy} {name of copyright owner}
+   Copyright 2016 Boundless, http://boundlessgeo.com
```
Comment (jodygarnett) [inline LICENSE]: thanks, looks great.

(Did all of this work happen in 2016? or is some of it 2015-2016...)

File: NOTICE [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Copyright ${year} ${name}
```
Comment (jodygarnett) [inline NOTICE]: think you want to get the $ out :)

File: build.gradle [hunk 3]
```diff
@@ -3,17 +3,21 @@
 buildscript {
     repositories {
         jcenter()
+        maven {
+          url "https://plugins.gradle.org/m2/"
+        }
```
Comment (jodygarnett) [inline build.gradle]: this change is unrelated to license headers.

friends don't let friends mix commits

File: spatialconnect/build.gradle [hunk 4]
```diff
@@ -70,3 +70,12 @@ dependencies {
     testCompile 'junit:junit:4.12'
     testCompile 'org.mockito:mockito-core:1.10.19'
 }
+
+license {
+    ext.year = Calendar.getInstance().get(Calendar.YEAR)
+    ext.name = 'Boundless http://boundlessgeo.com'
+    header rootProject.file('NOTICE')
```
Comment (jodygarnett) [inline spatialconnect/build.gradle]: Oh I see you are building the correct license notice as part of packaging step ... tricky

**Comment by marcusthebrown** [comment]
:+1:
